include/z64audio.h

```diff
@@ -104,20 +112,20 @@ typedef struct {
 typedef struct {
     /* 0x0 */ s16 delay;
     /* 0x2 */ s16 arg;
-} AdsrEnvelope; // size = 0x4
+} EnvelopePoint; // size = 0x4
 
 typedef struct {
     /* 0x00 */ u32 start;
     /* 0x04 */ u32 end;
     /* 0x08 */ u32 count;
-    /* 0x0C */ char unk_0C[0x4];
-    /* 0x10 */ s16 state[16]; // only exists if count != 0. 8-byte aligned
+    /* 0x0C */ u32 originalLength;
```

Comment:
iirc, this isn't used in OoT code and I'm not sure if this appears in any OoT soundfonts either? I know it does appear in MM soundfonts, could you explain this name a little and the context behind it?

Reply:
I'm actually not sure.  I want to say it came from a conversation with @zelda2774, but I'm not finding it in my searches.